api/app/src/providers/fitbit.ts

```diff
@@ -218,32 +218,45 @@ export class Fitbit extends Provider implements OAuth2 {
     return mapToBiometrics(date, breathing, cardio, hr, hrv, spo, tempCore, tempSkin);
   }
 
-  async fetchUserProfile(accessToken: string): Promise<FitbitUser> {
+  async fetchUserProfile(
+    accessToken: string,
+    params: { [k: string]: string | number } | undefined
+  ): Promise<FitbitUser> {
     return this.oauth.fetchProviderData<FitbitUser>(
       `${Fitbit.URL}/${Fitbit.API_PATH}/profile.json`,
       accessToken,
       async resp => {
         return fitbitUserResp.parse(resp.data);
-      }
+      },
+      params
     );
   }
 
-  async fetchWeights(accessToken: string, date: string): Promise<FitbitWeight> {
+  async fetchWeights(
+    accessToken: string,
+    date: string,
+    params: { [k: string]: string | number } | undefined
```

Comment:
this wont really work as intended, as the "en_GB" will get passed as a param *and* a header.

let's just add a new parameter, `extraHeaders`, that looks like this:

`extraHeaders?: { [k: string]: string }`